flink-python/docs/reference/pyflink.table/catalog.rst

```diff
@@ -250,3 +250,31 @@ A CatalogDescriptor is a template for creating a catalog instance. It closely re
     :toctree: api/
 
     CatalogDescriptor.of
+
+
+ObjectIdentifier
+----------------
+
+Identifies an object in a catalog. It allows to identify objects such as tables, views,
+function, or types in a catalog. An identifier must be fully qualified. It is the
```

Comment:
nit:
An identifier -> ObjectIdentifier
An identifier must be fully qualified. -> An identifier must be fully qualified, by this we mean .....